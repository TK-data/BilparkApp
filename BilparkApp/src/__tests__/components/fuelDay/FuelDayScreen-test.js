import React from 'react';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';

import FuelDayScreen from '../../../components/fuelDay/FuelDayScreen';

const initialState = {
};

const middlewares = [thunk];
const mockStore = configureStore(middlewares);

describe('Testing FuelDayScreen', () => {
  it('Renders as expected', () => {
    const comp = shallow(
      <FuelDayScreen />,
      { context: { store: mockStore(initialState) } },
    );
    expect(comp.dive()).toMatchSnapshot();
  });
});
