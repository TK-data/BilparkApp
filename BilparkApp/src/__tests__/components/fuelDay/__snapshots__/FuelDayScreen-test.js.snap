// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing FuelDayScreen Renders as expected 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgb(000, 039, 118)",
        "display": "flex",
        "flex": 2,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "display": "flex",
            "width": 750,
          }
        }
      >
        <Image
          resizeMode="contain"
          source={
            Object {
              "process": [Function],
            }
          }
          style={
            Object {
              "height": 200.1,
              "margin": 0,
              "marginTop": -10,
              "width": 112.5,
              "zIndex": 5,
            }
          }
        />
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "rgb(000, 039, 118)",
              "fontSize": 20,
              "marginBottom": 5,
              "marginTop": -15,
            }
          }
        >
          Pushvarsel
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flex": 2,
        }
      }
    >
      <Connect(FuelDayForm) />
    </View>
  </View>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <View
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "flex": 1,
              "justifyContent": "center",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "display": "flex",
                "width": 750,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={
                Object {
                  "process": [Function],
                }
              }
              style={
                Object {
                  "height": 200.1,
                  "margin": 0,
                  "marginTop": -10,
                  "width": 112.5,
                  "zIndex": 5,
                }
              }
            />
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "rgb(000, 039, 118)",
                  "fontSize": 20,
                  "marginBottom": 5,
                  "marginTop": -15,
                }
              }
            >
              Pushvarsel
            </Text>
          </View>
        </View>,
        <View
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "flex": 2,
            }
          }
        >
          <Connect(FuelDayForm) />
        </View>,
      ],
      "style": Object {
        "alignItems": "center",
        "backgroundColor": "rgb(000, 039, 118)",
        "display": "flex",
        "flex": 2,
        "justifyContent": "center",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "display": "flex",
                "width": 750,
              }
            }
          >
            <Image
              resizeMode="contain"
              source={
                Object {
                  "process": [Function],
                }
              }
              style={
                Object {
                  "height": 200.1,
                  "margin": 0,
                  "marginTop": -10,
                  "width": 112.5,
                  "zIndex": 5,
                }
              }
            />
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "rgb(000, 039, 118)",
                  "fontSize": 20,
                  "marginBottom": 5,
                  "marginTop": -15,
                }
              }
            >
              Pushvarsel
            </Text>
          </View>,
          "style": Object {
            "alignItems": "center",
            "display": "flex",
            "flex": 1,
            "justifyContent": "center",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Image
                resizeMode="contain"
                source={
                  Object {
                    "process": [Function],
                  }
                }
                style={
                  Object {
                    "height": 200.1,
                    "margin": 0,
                    "marginTop": -10,
                    "width": 112.5,
                    "zIndex": 5,
                  }
                }
              />,
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "rgb(000, 039, 118)",
                    "fontSize": 20,
                    "marginBottom": 5,
                    "marginTop": -15,
                  }
                }
              >
                Pushvarsel
              </Text>,
            ],
            "style": Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "display": "flex",
              "width": 750,
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "resizeMode": "contain",
                "source": Object {
                  "process": [Function],
                },
                "style": Object {
                  "height": 200.1,
                  "margin": 0,
                  "marginTop": -10,
                  "width": 112.5,
                  "zIndex": 5,
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "accessible": true,
                "allowFontScaling": true,
                "children": "Pushvarsel",
                "ellipsizeMode": "tail",
                "style": Object {
                  "color": "rgb(000, 039, 118)",
                  "fontSize": 20,
                  "marginBottom": 5,
                  "marginTop": -15,
                },
              },
              "ref": null,
              "rendered": "Pushvarsel",
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Connect(FuelDayForm) />,
          "style": Object {
            "alignItems": "center",
            "display": "flex",
            "flex": 2,
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": "View",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <View
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "white",
                  "display": "flex",
                  "width": 750,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={
                  Object {
                    "process": [Function],
                  }
                }
                style={
                  Object {
                    "height": 200.1,
                    "margin": 0,
                    "marginTop": -10,
                    "width": 112.5,
                    "zIndex": 5,
                  }
                }
              />
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "rgb(000, 039, 118)",
                    "fontSize": 20,
                    "marginBottom": 5,
                    "marginTop": -15,
                  }
                }
              >
                Pushvarsel
              </Text>
            </View>
          </View>,
          <View
            style={
              Object {
                "alignItems": "center",
                "display": "flex",
                "flex": 2,
              }
            }
          >
            <Connect(FuelDayForm) />
          </View>,
        ],
        "style": Object {
          "alignItems": "center",
          "backgroundColor": "rgb(000, 039, 118)",
          "display": "flex",
          "flex": 2,
          "justifyContent": "center",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "white",
                  "display": "flex",
                  "width": 750,
                }
              }
            >
              <Image
                resizeMode="contain"
                source={
                  Object {
                    "process": [Function],
                  }
                }
                style={
                  Object {
                    "height": 200.1,
                    "margin": 0,
                    "marginTop": -10,
                    "width": 112.5,
                    "zIndex": 5,
                  }
                }
              />
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "rgb(000, 039, 118)",
                    "fontSize": 20,
                    "marginBottom": 5,
                    "marginTop": -15,
                  }
                }
              >
                Pushvarsel
              </Text>
            </View>,
            "style": Object {
              "alignItems": "center",
              "display": "flex",
              "flex": 1,
              "justifyContent": "center",
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": Array [
                <Image
                  resizeMode="contain"
                  source={
                    Object {
                      "process": [Function],
                    }
                  }
                  style={
                    Object {
                      "height": 200.1,
                      "margin": 0,
                      "marginTop": -10,
                      "width": 112.5,
                      "zIndex": 5,
                    }
                  }
                />,
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "rgb(000, 039, 118)",
                      "fontSize": 20,
                      "marginBottom": 5,
                      "marginTop": -15,
                    }
                  }
                >
                  Pushvarsel
                </Text>,
              ],
              "style": Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "display": "flex",
                "width": 750,
              },
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "resizeMode": "contain",
                  "source": Object {
                    "process": [Function],
                  },
                  "style": Object {
                    "height": 200.1,
                    "margin": 0,
                    "marginTop": -10,
                    "width": 112.5,
                    "zIndex": 5,
                  },
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "accessible": true,
                  "allowFontScaling": true,
                  "children": "Pushvarsel",
                  "ellipsizeMode": "tail",
                  "style": Object {
                    "color": "rgb(000, 039, 118)",
                    "fontSize": 20,
                    "marginBottom": 5,
                    "marginTop": -15,
                  },
                },
                "ref": null,
                "rendered": "Pushvarsel",
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <Connect(FuelDayForm) />,
            "style": Object {
              "alignItems": "center",
              "display": "flex",
              "flex": 2,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": "View",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "context": Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
    },
  },
}
`;
